x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -47,22 +45,17 @@ export const getOperatorType = (entry: ExceptionEntry): OperatorType => {
  *
  * @param entry a single ExceptionItem entry
  */
-export const getExceptionOperatorSelect = (entry: ExceptionEntry): OperatorOption => {
-  const operatorType = getOperatorType(entry);
-  const foundOperator = EXCEPTION_OPERATORS.find((operatorOption) => {
-    return entry.operator === operatorOption.operator && operatorType === operatorOption.type;
-  });
-
-  return foundOperator ?? isOperator;
-};
-
-export const isEntryNested = (
-  tbd: ExceptionEntry | NestedExceptionEntry
-): tbd is NestedExceptionEntry => {
-  if (tbd.type === 'nested') {
-    return true;
+export const getExceptionOperatorSelect = (entry: Entry): OperatorOption => {
+  if (entriesNested.is(entry)) {
```

Comment:
@FrankHassanabad per your feedback, updated to use the `.is` built into io-ts types.